demo/vl-radio-group.html

```diff
@@ -11,12 +11,18 @@
 </head>
 
 <body>
-    <vl-demo-page data-vl-webcomponent="vl-radio-group"
-        data-vl-link="https://overheid.vlaanderen.be/webuniversum/v3/documentation/forms/vl-ui-radio">
-        <vl-demo data-vl-title="Radio group">
-            <vl-radio-group>
-                <vl-radio id="radio-1" data-vl-label="Ja" data-vl-value="yes"></vl-radio>
-                <vl-radio id="radio-2" data-vl-label="Nee" data-vl-value="no"></vl-radio>
+    <vl-demo-page data-vl-webcomponent="vl-radio-group" data-vl-link="https://overheid.vlaanderen.be/webuniversum/v3/documentation/forms/vl-ui-radio">
```

Comment:
Is er een reden waarom we in de demo overal beperken tot 2 radio's? In principe is dat een checkbox. Ik had destijds deze comment gemaakt bij Pieter en er waren bijkomende voorbeelden met vb 3 opties (die nu verdwenen zijn?).

De reden waarom is ook om aan te tonen dat de onderliggende implementatie OK is en niet enkel de 2 opties van elkaar laat afhangen.

Reply:
De demo pagina is zoveel als mogelijk een kopie van die van Webuniversum. Er is een unit test die de interne logica met meer dan 2 radio elementen valideert.